themes/default/content/what-is/what-is-yaml.md

```diff
@@ -0,0 +1,131 @@
+---
+title: What is YAML?
+meta_desc: |
+     YAML is a data serialization language that has steadily increased in popularity. Discover how to use YAML with Pulumi today.
+
+type: what-is
+page_title: "What is YAML?"
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+YAML is a data serialization language most commonly used for configuration files. Its easy readability and rich feature set have made it an increasingly popular choice over the years, for everything from configuration files to object serialization. Originally named "Yet Another Markup Language," the creators changed the name to "YAML Ain't a Markup Language" in order to better reflect its strength as a data-oriented language rather than simply markup.
+
+## How To Get Started With YAML
+
+Many of YAML's strongest features were inspired by other programming languages. Like Python, YAML uses whitespace indentation for defining the structure of your file. Strings, integers, floats, lists, and dictionaries are all natively supported, and it does also allow you to define custom data types. Dig far enough into the history of YAML and you'll find pieces of the PERL, C, and HTML specs.
+
+While YAML is frequently compared to JSON, it's important to note that the two are very closely related. YAML is actually a superset of JSON, and so it is capable of parsing JSON directly.
+
+The following is an example of YAML:
+
+```yaml
+---
+# An easy-to-read set of data on the Pulumi mascot, in YAML
+name: Pulumipus
+breed: platypus
+color: purple
+mascot: True
+age: 5
+hobbies:
+  - Kayaking
+  - Bouldering
+  - Reading
+  - Coding
+languages:
+  python: Expert
+    version: 3.7
+  typescript: Expert
+  csharp: Expert
+  java: Expert
+  yaml: Expert
+```
+
+The beginning of a YAML file is always three dashes (`---`) on the first line. From there, your file is built out of key-value pairs.
+
+```yaml
+name: Pulumipus
+breed: platypus
+color: purple
+```
+
+The first three key-value pairs are strings indicating that this creature is a purple platypus named Pulumipus, but YAML also supports integers, floats, and booleans to give their age and current status as a mascot:
+
+```yaml
+mascot: True
+age: 5
+```
+
+We also have access to lists (or arrays), indicated by preceding an indented item with a dash:
+
+```yaml
+hobbies:
+  - Kayaking
+  - Bouldering
+  - Reading
+  - Coding
+```
+
+You can even nest these key-value pairs for more granular information:
+
+```yaml
+languages:
+  python: Expert
+    version: 3.7
+  typescript: Expert
+  csharp: Expert
+  java: Expert
+  yaml: Expert
+```
+
+## The Benefits of YAML With Pulumi
```

Comment:
in the same way you went through and explained lists and key value pairs on the platypus data list example above. I think you should go through and explain name, runtime, resources, and outputs in this section, so at least they walk away with a bit of knowledge of Pulumi here